change format name

leLuan1102 opened the pull request:
In đậm đầu dòng

leLuan1102 committed:
change name

nhattien2211 commented on index.html:
```diff
@@ -22,11 +22,11 @@ <h2 id="header-introduction"><em>Keep track of my website for seeing more future
+                        <li> <strong>Age     :</strong> 25 years old</li>
+                        <li><strong>Job      :</strong> Electrical & Electronics・Automation Engineer</li>
+                        <li> <strong>Location:</strong> Ho Chi Minh City, Viet Nam</li>
+                        <li><strong>Me here  :</strong>  <a href="https://www.facebook.com/tien.lenhat/" target="_blank">Facebook</a></li>
```
lost the text "Contact" in "Contact me"

nhattien2211 replied:
lost the text "Contact" in "Contact me"

nhattien2211 commented on index.html:
```diff
@@ -39,11 +39,11 @@ <h2 class="content-heading">・About me</h2>
-                        <li>Hydraulic and Pneumatic Simulation: Festo FluidSIM</li>
-                        <li>Microcontroller Programming: Keil C</li>
-                        <li>Mechanical Desgin: SolidWorks</li>
+                        <li><strong>Plc and SCADA  :</strong>  TIA Portal V13</li>
```
PLC stands for Programmable Logic Controller so it should be all letters uppercased (PLC)

nhattien2211 reviewed (changes requested):
Please check line 29 and line 42

nhattien2211 committed:
Update index.html

Line 29: Change "Me here" into "Contact me here"
Line 42: Change "Plc" into "PLC"

nhattien2211 commented:
the fix-code is okay.
